xbmc/video/VideoInfoTag.cpp

```diff
@@ -591,6 +598,11 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
   {
     m_fRating = m_fRating / max_value * 10; // Normalise the Movie Rating to between 1 and 10
   }
+  const TiXmlElement* urElement = movie->FirstChildElement("userrating");
```

Comment:
No it's not. Because we get the value for userrating initially a few lines before this. But to do the normalise and enter this code block, the max is needed I think? So this will not normalise to ten when there is no max given.
The documentation for ```QueryIntArttribute``` says 
```If the attribute does not exist, then TIXML_NO_ATTRIBUTE is returned.``` this will lead to the if getting ignored